saleor/app/models.py

```diff
@@ -103,6 +108,22 @@ class AppToken(models.Model):
     auth_token = models.CharField(default=generate_token, unique=True, max_length=30)
 
 
+class AppExtension(models.Model):
+    app = models.ForeignKey(App, on_delete=models.CASCADE, related_name="extensions")
+    label = models.CharField(max_length=256)
+    url = models.URLField()
+    view = models.CharField(choices=AppExtensionView.CHOICES, max_length=128)
+    type = models.CharField(choices=AppExtensionType.CHOICES, max_length=128)
+    mounting_place = models.CharField(
```

Comment:
I'm not convinced about this name, maybe `target` would be a good name?

Reply:
Do you mean - replacing `mounting_place` to `target` or `target_place`?

Reply:
I mean just `target` and `AppExtensionTarget`.